confdb/aclocal_f77.m4

```diff
@@ -800,17 +824,17 @@ AC_LANG_PUSH([C])
 AC_COMPILE_IFELSE([
     AC_LANG_PROGRAM([],[long long a;])
 ],[
-    AC_DEFINE(HAVE_LONG_LONG,1,[Define if long long allowed])
-])
```

Comment:
User/device may pass-in options to disable "long long". All `HAVE_type` macros are defined by `config_filter`; `configure.ac` only need check `sizeof`.